src/corg/developer-environment-setup/intellij-idea-setup.md

```diff
@@ -86,7 +86,7 @@ Then, you need to configure it using the [tools folder of the top-level reposito
 
 - Enable the plugin
 - Import Eclipse Java Formatter config file from the tools folder of Nuxeo source code
-  `nuxeo/tools/nuxeo_formatter.xml`.
+  `nuxeo/tools/nuxeo-formatter.xml`.
```

Comment:
Hi @semisse , where did you find the change? As far as I see, the file is still `nuxeo_formatter.xml` in the Nuxeo repository <https://github.com/nuxeo/nuxeo/blob/master/tools/nuxeo_formatter.xml>

Reply:
Hello @mincong-h , you are 100% right. I was confused with the javascript-formatter.xml. Apologies.

Reply:
No problem :)